giap.client/vite.config.ts

```diff
@@ -0,0 +1,61 @@
+import { fileURLToPath, URL } from 'node:url';
+
+import { defineConfig } from 'vite';
+import plugin from '@vitejs/plugin-react';
+import fs from 'fs';
+import path from 'path';
+import child_process from 'child_process';
+import { env } from 'process';
+
+const baseFolder =
```

Comment:
Rather use `.env` files with environment specific API endpoints. Like used here: https://github.com/privacybydesign/yivi-portal/blob/f1278ad7728602e060dd6ef8ba33834b206b1716/portal_spa/src/services/axiosInstance.ts#L8
Also https is not needed on localhost.